include/deal.II/numerics/error_estimator.h

```diff
@@ -308,7 +333,8 @@ class KellyErrorEstimator
                         const Function<spacedim>     *coefficients   = 0,
                         const unsigned int       n_threads = numbers::invalid_unsigned_int,
                         const types::subdomain_id subdomain_id = numbers::invalid_subdomain_id,
-                        const types::material_id       material_id = numbers::invalid_material_id);
+                        const types::material_id       material_id = numbers::invalid_material_id,
+                        const KellyErrorEstimator<dim,spacedim>::Strategy strategy = cell_diameter_over_24);
```

Comment:
please document the new argument in the documentation of this function (and of all other functions that are not simply referring to this one in their documentation texts).